appconfiguration/azure-appconfiguration-sample/pom.xml

```diff
@@ -10,10 +10,10 @@
   </parent>
   <modelVersion>4.0.0</modelVersion>
 
-  <artifactId>azure-appconfiguration-sample</artifactId>
   <groupId>com.azure.spring</groupId>
-  <version>2.0.0</version> <!-- {x-version-update;com.azure.spring:azure-appconfiguration-sample;dependency} -->
-  <name>Spring Cloud Azure App Configuration Sample</name>
+  <artifactId>azure-appconfiguration-sample</artifactId>
+  <version>1.0.0</version>
+  <name>Load Configuration Properties From App Configuration to Spring Environment</name>
```

Comment:
To

Reply:
Same 
![image](https://user-images.githubusercontent.com/4465723/148524596-70e32b53-3ab5-464a-b06b-65c196202148.png)

https://capitalizemytitle.com/